Author is OpenStax test is failing

The author for https://staging.cnx.org/contents/yqV9q0HH@13.1:plLt9FYr@6/Introduction is still `OpenStax College` rather than `OpenStax`.

This is causing the author test at `tests/webview/ui/test_content.py::test_author_is_openstax` to fail?

<details><summary>failure message (click to expand)</summary>

```
webview_base_url = 'https://staging.cnx.org'
selenium = <selenium.webdriver.chrome.webdriver.WebDriver (session="3a3af2c809fda4b85db1ff18e8ffe2ce")>

    @markers.webview
    @markers.test_case('C132542')
    @markers.nondestructive
    def test_author_is_openstax(webview_base_url, selenium):
        # GIVEN the home page and a book
        home = Home(selenium, webview_base_url).open()
        book = home.featured_books.openstax_list[0]
    
        # WHEN the book's cover is clicked
        content = book.click_book_cover()
    
        # THEN the displayed author is OpenStax
        content_header = content.content_header
        assert content_header.is_book_by_displayed
        assert content_header.are_authors_displayed
>       assert content_header.authors == 'OpenStax'
E       AssertionError: assert 'OpenStax College' == 'OpenStax'
E         - OpenStax College
E         + OpenStax

tests/webview/ui/test_content.py:156: AssertionError
```
</details>

---

Is there any plans to fix the author or should we adjust the test?

--- a/tests/webview/ui/test_content.py
+++ b/tests/webview/ui/test_content.py
@@ -141,7 +141,7 @@ def test_navs_and_elements_are_displayed(webview_base_url, selenium):
 @markers.webview
 @markers.test_case('C132542')
 @markers.nondestructive
-def test_author_is_openstax(webview_base_url, selenium):
+def test_author_contains_openstax(webview_base_url, selenium):
     # GIVEN the home page and a book
     home = Home(selenium, webview_base_url).open()
     book = home.featured_books.openstax_list[0]
@@ -153,7 +153,7 @@ def test_author_is_openstax(webview_base_url, selenium):
     content_header = content.content_header
     assert content_header.is_book_by_displayed
     assert content_header.are_authors_displayed
-    assert content_header.authors == 'OpenStax'
+    assert 'OpenStax' in content_header.authors
 
 
 @markers.webview

